config/routes.rb

```diff
@@ -2014,6 +2014,9 @@
 
   root :to => 'dashboard#login'
 
+  # Let's serve pictures directly from the DB
+  get '/pictures/:basename' => 'picture#show', :basename => /[\da-zA-Z]+\.[\da-zA-Z]+/
```

Comment:
You can probably just use \w...you do get `_` chars as well, but it shouldn't matter, and it reads nicer: `:basename => /\w+\.\w+/`